schema.sql

```diff
@@ -0,0 +1,175 @@
+                                                    `name` VARCHAR(45) NOT NULL,
+                                                    `password` VARCHAR(45) NOT NULL,
+                                                    `contacts` VARCHAR(255) NOT NULL,
+                                                    PRIMARY KEY (`iser_id`),
+                                                    UNIQUE INDEX `email_UNIQUE` (`email` ASC) VISIBLE)
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`author_item`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`author_item` (
+                                                           `author_id` INT NOT NULL AUTO_INCREMENT,
+                                                           `user_id` INT NOT NULL,
+                                                           `item_id` INT NOT NULL,
+                                                           PRIMARY KEY (`author_id`),
+                                                           INDEX `user_idx` (`user_id` ASC) VISIBLE,
+                                                           INDEX `item_idx` (`item_id` ASC) VISIBLE,
+                                                           CONSTRAINT `user`
+                                                               FOREIGN KEY (`user_id`)
+                                                                   REFERENCES `yeticave_db`.`user` (`iser_id`)
+                                                                   ON DELETE CASCADE
+                                                                   ON UPDATE CASCADE,
+                                                           CONSTRAINT `item`
+                                                               FOREIGN KEY (`item_id`)
+                                                                   REFERENCES `yeticave_db`.`item` (`item_id`)
+                                                                   ON DELETE CASCADE
+                                                                   ON UPDATE CASCADE)
+    ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `yeticave_db`.`winner_item`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `yeticave_db`.`winner_item` (
+                                                           `winner_id` INT NOT NULL AUTO_INCREMENT,
+                                                           `user_id` INT NOT NULL,
+                                                           `item_id` INT NOT NULL,
+                                                           PRIMARY KEY (`winner_id`),
+                                                           INDEX `user_idx` (`user_id` ASC) VISIBLE,
+                                                           INDEX `item_idx` (`item_id` ASC) VISIBLE,
```

Comment:
[ERROR in query 12] You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'VISIBLE,
                                                             INDEX `ite' at line 8